schemas/openapi_schema.json

```
@@ -2926,736 +4048,839 @@
+            "x-optimade-support": "optional",
+            "x-optimade-queryable": "optional"
+          },
+          "space_group_hall": {
```

```suggestion
          "space_group_symbol_hall": {
```
As per specification